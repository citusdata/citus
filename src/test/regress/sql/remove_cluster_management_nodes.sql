-- When needed in some tests, GetLocalNodeId() does not behave correctly.
SELECT citus_remove_node('localhost', 8887);
SELECT citus_remove_node('localhost', 9995);
SELECT citus_remove_node('localhost', 9992);
SELECT citus_remove_node('localhost', 9998);
SELECT citus_remove_node('localhost', 9997);
SELECT citus_remove_node('localhost', 8888);
