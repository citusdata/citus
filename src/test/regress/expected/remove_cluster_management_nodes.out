-- When needed in some tests, GetLocalNodeId() does not behave correctly.
SELECT citus_remove_node('localhost', 8887);
 citus_remove_node
---------------------------------------------------------------------

(1 row)

SELECT citus_remove_node('localhost', 9995);
 citus_remove_node
---------------------------------------------------------------------

(1 row)

SELECT citus_remove_node('localhost', 9992);
 citus_remove_node
---------------------------------------------------------------------

(1 row)

SELECT citus_remove_node('localhost', 9998);
 citus_remove_node
---------------------------------------------------------------------

(1 row)

SELECT citus_remove_node('localhost', 9997);
 citus_remove_node
---------------------------------------------------------------------

(1 row)

SELECT citus_remove_node('localhost', 8888);
 citus_remove_node
---------------------------------------------------------------------

(1 row)

